config/sync/core.entity_view_display.node.stanford_page.teaser.yml

```diff
@@ -0,0 +1,125 @@
+uuid: ee2db7f1-aa45-4ac8-b854-baea77a408ca
+langcode: en
+status: false
+dependencies:
+  config:
+    - core.entity_view_mode.node.teaser
+    - field.field.node.stanford_page.layout_builder__layout
+    - field.field.node.stanford_page.layout_selection
+    - field.field.node.stanford_page.stanford_intranet__access
+    - field.field.node.stanford_page.su_basic_page_type
+    - field.field.node.stanford_page.su_page_banner
+    - field.field.node.stanford_page.su_page_components
+    - field.field.node.stanford_page.su_page_description
+    - field.field.node.stanford_page.su_page_image
+    - node.type.stanford_page
+  module:
+    - ds
+    - field_formatter_class
+    - layout_builder
+    - layout_builder_restrictions
+    - layout_library
+    - stanford_media
+    - user
+third_party_settings:
+  layout_builder:
+    allow_custom: false
+    enabled: false
+  layout_library:
+    enable: true
+  layout_builder_restrictions:
+    entity_view_mode_restriction:
+      allowed_blocks:
+        'Chaos Tools': {  }
+        'Content fields':
+          - 'field_block:node:stanford_page:su_page_banner'
+          - 'field_block:node:stanford_page:su_page_components'
+          - 'field_block:node:stanford_page:title'
+        Forms: {  }
+        Help: {  }
+        Menus:
+          - 'menu_block:footer'
+          - 'system_menu_block:footer'
+          - 'menu_block:main'
+          - 'system_menu_block:main'
+        'SimpleSAMLphp Authentication': {  }
+        'Stanford SimpleSAML PHP': {  }
+        System: {  }
+        'User fields': {  }
+        core: {  }
+      allowed_layouts:
+        - jumpstart_ui_one_column
+        - jumpstart_ui_two_column
+        - jumpstart_ui_three_column
+      whitelisted_blocks: {  }
+      blacklisted_blocks: {  }
+    allowed_block_categories: {  }
+  ds:
+    layout:
+      id: pattern_basic-page-types-vertical-teaser
+      library: null
+      disable_css: false
+      entity_classes: all_classes
+      settings:
+        pattern:
+          field_templates: default
+    regions:
+      basic_page_types_category:
+        - su_basic_page_type
+      basic_page_types_title:
+        - node_title
+      basic_page_types_description:
+        - su_page_description
+      basic_page_types_page_image:
+        - su_page_image
```

Comment:
Do these need to update now?

Reply:
actually, I dont think we need this config.  we're not using the teaser display mode for basic pages; we just use the card display mode right?

Reply:
I'm checking now on local.

Reply:
yeah, it's not necessary.  Removed this config.